{
    "type": "bundle",
    "id": "bundle--389903e5-0b2d-46d7-9f88-3a7cf99a662a",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--bd431643-1b5e-440f-969e-aa588053ce26",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2022-11-19T00:05:32.185559Z",
            "modified": "2022-11-19T00:05:32.185559Z",
            "name": "CVE-2022-41893",
            "description": "TensorFlow is an open source platform for machine learning. If `tf.raw_ops.TensorListResize` is given a nonscalar value for input `size`, it results `CHECK` fail which can be used to trigger a denial of service attack. We have patched the issue in GitHub commit 888e34b49009a4e734c27ab0c43b0b5102682c56. The fix will be included in TensorFlow 2.11. We will also cherrypick this commit on TensorFlow 2.10.1, 2.9.3, and TensorFlow 2.8.4, as these are also affected and still in supported range.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-41893"
                }
            ]
        }
    ]
}